packages/docs/stories/templates/form.stories.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import type { Meta } from '@storybook/web-components';
+import { Title, Subtitle, Description, Stories } from '@storybook/blocks';
+import { html } from 'lit';
+
+const meta: Meta = {
+  parameters: {
+    docs: {
+      page: () => (
+        <>
+          <Title />
+          <Subtitle />
+          <Description />
+          <Stories />
+        </>
+      ),
+    },
+  },
+  title: 'Templates/Forms',
+};
+export default meta;
+
+export const ContactForm = {
+  render: () => html`
+    <style>
+    .synergy-form-demo {
+      background: var(--syn-color-neutral-0);
+      border: 1px dashed var(--syn-color-neutral-300);
+      margin: 0 auto;
+      padding: 10px 20px;
+      max-width: 750px;
+      min-width: 360px;
+    }
+
+    .synergy-form-demo h1 {
```

Comment:
1. What do you think about wrapping all templates in iFrames on the docs page, so you can get rid of all classes to scope e. g. `h1` –  here an example for implementation https://github.com/storybookjs/storybook/issues/13765#issuecomment-1538517811.
2. We implemented PostCSS to look into html`` template literals: https://github.com/solid-design-system/solid/commit/452ac1c4ba1f37f304d056c033541779227ca406#diff-56a2517c7d913d792b087e343de1d7e6e9e23e493608be111aa8ff69dd4bafe8 This made it possible to do SCSS nesting like e. g. here https://github.com/solid-design-system/solid/commit/452ac1c4ba1f37f304d056c033541779227ca406#diff-06a0efe585bedb897c7a0062a74ce5d5aa6a058e3406ecb973b2eddef1bf952aR145-R161

I think this could be interesting for us?

Reply:
I will have a look at the iframe solution. What do you think about using css nesting directly? We would not need to have to use a toolchain for that, as support seems pretty good nowadays: https://caniuse.com/css-nesting. Safari would be supported from MacOS Sonoma and i(pad) OS 17 onwards.

I just do not want to integrate more tooling than necessary.

Reply:
iFrame encapsulation works great, thanks for the tip! Unfortunately, you are only allowed to set the iframe to a fixed size, no auto sizing is supported :(. I have opted to set the height to the current forms height manually. Not an elegant solution, but I think this is fine for now.

Reply:
> I just do not want to integrate more tooling than necessary.

That sounds very good.

> CSS Nesting

There are quite some drawbacks ATM. I fear that this might be tooooo progressive. 😆

I think the iFrame solution already improves readability a lot, therefore I think that's totally fine like that.